### PR TITLE
[Ready]Newniforms

### PR DESCRIPTION
**Pull request by Kmc2000**
Changes the uniform styles to black!

The worn states are fully tested and working.

Some jobs do NOT have these uniforms, as i feel they're fine as is.

Bartender, miner, clown, mime.

Screens:

 XO

CAP

QM

BOTANY

CE

Shitsec (a beret will look way cooler with this, but stock sec looks like that)

CMO

Legend: (for those who care):

1 stripe on leg = uncomissioned officer
2 stripes on legs = comissioned officer
no stripes = junior officer/civilian

1 pip (gold thing on the collar) = Lieutenant
2 Pips = Departmental Head
3 Pips = Ship captain

This is pretty much what all the item states are.

and yes, they can ALL be worn casually as well

:cl: Kmc2000
imageadd: Changed most of the uniform sprites, check em out

/:cl:

### CODE REVIEW

**Comment by Kmc2000** [comment]
Yes i know commits. Had to reset my branch from the LAST pr..shudder

**Comment by ike709** [comment]
Seriously how do you keep doing that...?

File: ftl13.dme [hunk 1]
```diff
@@ -972,7 +972,6 @@
 #include "code\modules\atmospherics\machinery\components\unary_devices\vent_pump.dm"
 #include "code\modules\atmospherics\machinery\components\unary_devices\vent_scrubber.dm"
 #include "code\modules\atmospherics\machinery\other\meter.dm"
-#include "code\modules\atmospherics\machinery\other\miner.dm"
```
Comment (monster860) [inline ftl13.dme]: Uhhhh..............................................

Comment (Kmc2000) [inline ftl13.dme]: SHHHH

**Comment by Kmc2000** [comment]
d-d-d-d-d-aaaamn

Thanks git experts :tm:

**Comment by Vivalas** [comment]
:pray:

**Pull request merged**